### DISCUSSION TITLE
Search Call panel optimizations

### DISCUSSION THREAD

**Discussion by canuckcam** [chuot/rdio-scanner#91] [Ideas]
Could columns for Radio IDs (alias if exists) and the duration of the call be added in the Search Call panel in desktop browser and landscape mode? And maybe a config option for # of calls per page (eg. 50)

It would make it easier to look for specific calls - I do understand that the existing UI is built for mobile devices as it works well on my iPhone and Android. 

If the Next/Prev and search parameter buttons are in a persistent panel on the bottom of the screen such that it doesn't scroll off the screen, that would make viewing bulk search results easier?

No idea if this makes sense - I don't know how to do any of this, just critiquing, the easiest job. 🗡️ 😄

**Comment by chuot** [comment]
Those are things I will look at later when everything will settle down with v6. Thanks for the idea.
